Fixtures/MixedTargets/BasicMixedTargets/MixedTargetWithCustomPaths/Sources/OldCar.m

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+
+// All three import statements should be supported.
```

Comment:
What about `#import  <Public/OldCar.h>`?

Reply:
Yes, that works too. The path within `<...>` seems to fallback to evaluating as a `"..."` based on the user header search paths passed to the compiler. I added a few examples in eae602c0c87267353296cfa4387a1398708f3f4b to demonstrate.